mimesis/builtins/ru.py

```diff
@@ -82,7 +82,7 @@ def series_and_number(self) -> str:
         :Example:
             57 16 805199.
         """
-        return '{}{}'.format(
+        return '{} {}'.format(
```

Comment:
Please, update test for this method.

Reply:
Mmm, can you explane what do you mean?
I am new for that.
I saw test at https://github.com/lk-geimfari/mimesis/blob/master/tests/test_builtins/ru/test_russia_spec.py
'''
def test_series_and_number(russia):
    result = russia.series_and_number()
    assert result is not None
'''
The test does not require updating after my edit